src/app/modules/group/pages/my-groups/my-groups.component.scss

```diff
@@ -0,0 +1,25 @@
+@import 'src/colors.scss';
+
+.group-info-container {
+  .group-info-header {
+    display: flex;
+    flex-direction: column;
+    margin-top: 2.5rem;
+    margin-bottom: 2.5rem;
+    color: $base-text-color;
+
+    .group-title {
+      font-size: 2rem;
+      margin-bottom: 1rem;
+      color: #4A4A4A;
+    }
+
+    .group-subtitle {
+      font-size: 1.2rem;
+    }
```

Comment:
Quite common to almost all page, could you try to make it reusable?

Reply:
I have created page-header scss component and it was integrated only for my groups page. Integration it for whole app probably requires separated task